_blog.yml

```
@@ -3692,5 +3692,12 @@
     - llm
     - inference
     
-        
+- local: policy-blog
+  title: "Public Policy at Hugging Face"
+  author: irenesolaiman
+  thumbnail: /blog/assets/151_policy_ntia_rfc/us_policy_thumbnail.png
```

```suggestion
  thumbnail: /blog/assets/policy_docs/policy_blog_thumbnail.png
```

(the US-centric version was still referenced here :))

---

oh great catch!!